src/main/java/hudson/plugins/parameterizedtrigger/SubProjectData.java

```diff
@@ -44,23 +44,23 @@
  */
 public class SubProjectData {
 
-    private final Comparator customComparator = new Comparator<AbstractProject>() {
-        public int compare(AbstractProject abstractProject1, AbstractProject abstractProject2) {
-            return abstractProject1.getFullName().compareTo(abstractProject2.getFullName());
+    private final Comparator<Job> customComparator = new Comparator<Job>() {
+        public int compare(Job job1, Job job2) {
+            return job1.getFullName().compareTo(job2.getFullName());
         }
     };
 
-    private final Set<AbstractProject> dynamic = new TreeSet<AbstractProject>(customComparator);
-    private final Set<AbstractProject> fixed = new TreeSet<AbstractProject>(customComparator);
-    private final Set<AbstractProject> triggered = new TreeSet<AbstractProject>(customComparator);
+    private final Set<Job> dynamic = new TreeSet<>(customComparator);
+    private final Set<Job> fixed = new TreeSet<>(customComparator);
+    private final Set<Job> triggered = new TreeSet<>(customComparator);
     private final Set<String> unresolved = new TreeSet<String>();
 
     /**
      * A set intended to hold dynamically configured sub project
      *
      * @return  A set reserved for dynamically configured sub project
      */
-    public Set<AbstractProject> getDynamic() {
+    public Set<Job> getDynamic() {
```

Comment:
It breaks binary compatibility. Would be better to introduce new methods

Reply:
Looks fine as it's probably only use internally.